### ISSUE TITLE
Typescript error: Cannot find type definition file for 'node'

### ISSUE THREAD

**Issue by ItsNoHax** [getsentry/sentry-react-native#1132]
Similar to: https://github.com/getsentry/sentry-javascript/issues/2927

Node types are missing and fail my ```tsc``` step.
Error:
```
$ tsc
node_modules/@sentry/hub/dist/interfaces.d.ts:1:23 - error TS2688: Cannot find type definition file for 'node'.

1 /// <reference types="node" />
                        ~~~~

node_modules/@sentry/hub/dist/interfaces.d.ts:3:25 - error TS2307: Cannot find module 'domain'.

3 import * as domain from 'domain';
                          ~~~~~~~~

Found 2 errors.

```
As a temporary workaround I tried adding @types/node to my own devDependencies but it started overriding react-native methods and gave other problems.

 "@sentry/react-native": "^1.9.0",

**Comment by jennmueng** [comment]
@ItsNoHax My colleagues over at @getsentry/javascript are aware of the node types issue and are currently working to fix it. I'll include the fix in a new release and let you know once I hear anything from them. 

What's unusual however, is `Span` shouldn't even be exported from the react native package, nor even exported from `@sentry/core` are you getting the package from some branch/fork?

**Comment by ItsNoHax** [comment]
Hi @jennmueng ,
Great to hear!

Not that i'm aware of, I installed it using: ```yarn add @sentry/react-native```

**Comment by kamilogorek** [comment]
Thanks for letting us know. Cheers!

**Comment by ItsNoHax** [comment]
Why did you close the issue @kamilogorek? The original error of node-types is still happening.

**Comment by kamilogorek** [comment]
Got it, my bad. Read your last comment as resolved. Reopened.

**Comment by ItsNoHax** [comment]
Hi @jennmueng @kamilogorek , I see the bug has been fixed upstream. Any ETA on when it will be fixed in this package?

**Comment by jennmueng** [comment]
@ItsNoHax This fix should be included in [2.0.1](https://github.com/getsentry/sentry-react-native/releases/tag/2.0.1).

**Comment by ItsNoHax** [comment]
Can confirm it's fixed. Thanks a lot @jennmueng
